examples/topo_deep_neural_networks.ipynb

```diff
@@ -0,0 +1,277 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Toplogy of Deep Neural Networks\n",
+    "\n",
+    "This notebook will show you how easy it is to use gdeep to reproduce the experiments of the paper [Topology of Deep Neural Networks](https://arxiv.org/pdf/2004.06093.pdf), by Naizat et. al. In this work, the authors studied the evolution of the topology of a dataset as embedded in the successive layers of a Neural Network, trained for classification on this dataset.\n",
+    "\n",
+    "Their main findings can be summarized as follows:\n",
+    "\n",
+    "- Neural networks tend to simplify the topology of the dataset accross layers.\n",
+    "\n",
+    "- This decrease in topological complexity is more efficient when the activation functions are non-homeomorphic, as it is the case for ReLu or leakyReLu.\n",
+    "\n",
+    "Here is an illustration from the paper:\n",
+    "\n",
+    "![img](./images/topology_accross_layers.png)\n",
```

Comment:
Force the image addition via `git add -f /examples/images/topology_across_layers.png`